# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ArmActionActionGoal action_goal
ArmActionActionResult action_result
ArmActionActionFeedback action_feedback
